src/ontology/mondo-edit.obo

```diff
@@ -362844,7 +362844,7 @@ property_value: seeAlso https://rarediseases.info.nih.gov/diseases/10353/porphyr
 [Term]
 id: MONDO:0019143
 name: angiostrongyliasis
-def: "and that is acquired by the ingestion of the infective larvae on vegetables or in raw or undercooked snails, slugs, land crabs, freshwater shrimps, frogs and lizards. The main feature is eosinophilic meningitis, with clinical manifestations including fever, headache, malaise, fatigue, vomiting, rhinorrhea, blurred vision, diplopia, cough, stiff neck, enteritis, constipation and paraesthesia due to the movement of the worms from the intestines to the lungs, central nervous system and eyes. In severe cases without treatment, coma and death can occur." [Orphanet:74]
+def: "And that is acquired by the ingestion of the infective larvae on vegetables or in raw or undercooked snails, slugs, land crabs, freshwater shrimps, frogs and lizards. The main feature is eosinophilic meningitis, with clinical manifestations including fever, headache, malaise, fatigue, vomiting, rhinorrhea, blurred vision, diplopia, cough, stiff neck, enteritis, constipation and paraesthesia due to the movement of the worms from the intestines to the lungs, central nervous system and eyes. In severe cases without treatment, coma and death can occur." [Orphanet:74]
```

Comment:
The def should be:

A foodborne zoonotic disease, endemic to Southeast Asia and the Pacific Islands, caused by the rat lungworm Angiostrongylus cantonensis and that is acquired by the ingestion of the infective larvae on vegetables or in raw or undercooked snails, slugs, land crabs, freshwater shrimps, frogs and lizards. The main feature is eosinophilic meningitis, with clinical manifestations including fever, headache, malaise, fatigue, vomiting, rhinorrhea, blurred vision, diplopia, cough, stiff neck, enteritis, constipation and paraesthesia due to the movement of the worms from the intestines to the lungs, central nervous system and eyes. In severe cases without treatment, coma and death can occur.